test/fixtures/typescript/ts/test-callsite.ts

```diff
@@ -0,0 +1,9 @@
+const { getCallSite } = require('node:util');
+const { mapCallSite } = require('node:module');
+
+interface CallSite {
+    A;
+    B;
```

Comment:
It looks like our linter and formatter doesn't properly format `.ts` files:

```suggestion
  A;
  B;
```